apps/remix-ide/src/app/editor/examples.js

```diff
@@ -248,10 +248,92 @@ contract BallotTest {
+    }
+  })()`
+
+var deployWithEthers = `(async function() {
+    try {
+          console.log('Running deployWithEthers script...')
+      
+          const contractName = 'Storage' // Change this for other contract
+          const constructorArgs = []    // Put constructor args (if any) here for your contract
+  
+          // Note that the script needs the ABI which is generated from the compilation artifact.
+          // Make sure contract is compiled and artifacts are generated
+          const artifactsPath = \`browser/contracts/artifacts/\${contractName}.json\` // Change this for different path
+        
+          const metadata = JSON.parse(await remix.call('fileManager', 'getFile', artifactsPath))
+          // 'web3Provider' is a remix global variable object
+          const signer = (new ethers.providers.Web3Provider(web3Provider)).getSigner()
+        
+          let factory = new ethers.ContractFactory(metadata.abi, metadata.data.bytecode.object, signer);
+        
+          let contract = await factory.deploy(...constructorArgs);
+        
+          console.log('Contract Address: ', contract.address);
+        
+          // The contract is NOT deployed yet; we must wait until it is mined
+          await contract.deployed()
+          console.log('Deployment successful.')
+    } catch (e) {
+          console.log(e.message)
+    }
+})()`
+
+var readme = `REMIX EXAMPLE PROJECT
+
+Remix example project is the project which is there at the time of very first loading of Remix. It contains mainly 3 directories:
+'contracts', 'scripts' & 'tests'.
+
+'contracts' directory holds 3 contracts with different complexity level.
+'scripts' directory holds the scripts to deploy a contract using web3.js or ethers.js libraries.
+'tests' directory contains a test file with unit tests for 'Ballot' contract.
+
+DEPLOY CONTRACT
+
+A contract can be deployed using the scripts in 'scripts' folder. Currently, they are configured to deploy the 'Storage' contract.
+For deployment of any other contract, 'contractName' and 'constructorArgs' should be update (along with other code if required).
+
+To run a script, just right click on file name and click 'Run'.
+
+Consoles from script will appear in remix terminal.
+`
 
 module.exports = {
-  storage: { name: '1_Storage.sol', content: storage },
-  owner: { name: '2_Owner.sol', content: owner },
-  ballot: { name: '3_Ballot.sol', content: ballot },
-  ballot_test: { name: 'tests/4_Ballot_test.sol', content: ballotTest }
+  storage: { name: 'contracts/1_Storage.sol', content: storage },
+  owner: { name: 'contracts/2_Owner.sol', content: owner },
+  ballot: { name: 'contracts/3_Ballot.sol', content: ballot },
+  deployWithWeb3: { name: 'scripts/deployWithWeb3.js', content: deployWithWeb3 },
+  deployWithEthers: { name: 'scripts/deployWithEthers.js', content: deployWithEthers },
+  ballot_test: { name: 'tests/4_Ballot_test.sol', content: ballotTest },
+  readme: { name: 'README', content: readme }
```

Comment:
in my test branch, I had `call.js` and `transact.js` , would it make sense to put it here too?

Reply:
yes definitely. May be we can have a separate PR for that.